### PR TITLE
Add contributors portal and other links

### PR DESCRIPTION
**Pull request by eviau-artefactual**
These changes should help a new contributor find their way in our documentation.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # Artefactual
 
 For more information about us, please [visit artefactual.com](https://www.artefactual.com/).
+
+## Contributing
+
+We are actively developing the [Contributors portal](https://contributors.artefactual.com) to host guidelines and information
+related to contributing to AtoM, Archivematica, and the dependencies which we maintain.
```
Comment (fiver-watson) [inline profile/README.md]: we don't maintain the dependencies so i might suggest rewording this to something like, 

* "and their dependencies" or
* "as well as listing their dependencies"

Also you mention Enduro on the actual portal, but not here - not sure if that's intentional or not? One option is simply to not list projects at all (since other applications like AIPScan etc could end up here as well), and just say "the projects that Artefactual maintains" or something

File: profile/README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+## Related organizations
+
+Most of our projects are hosted at one of the following:
+
+- [Main Artefactual repo](https://www.github.com/artefactual), this page
+- [Artefactual labs](https://www.github.com/artefactual-labs), smaller projects and experiments
+- [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects
```
Comment (fiver-watson) [inline profile/README.md:16]: if you wanted you could add [Artefactual-SDPS](https://github.com/artefactual-sdps) for projects related to Enduro

**Review by fiver-watson — COMMENTED** [review]
A couple suggestions for your consideration!

File: profile/README.md (hunk 1 above)
Comment (eviau-artefactual) [inline profile/README.md]:
```suggestion
related to contributing to the projects that Artefactual maintains.
```

File: profile/README.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+We are actively developing the [Contributors portal](https://contributors.artefactual.com) to host guidelines and information
+related to contributing to the projects that Artefactual maintains.
+
+## Related organizations
+
+Most of our projects are hosted at one of the following:
+
+- [Main Artefactual repo](https://www.github.com/artefactual), this page
+- [Artefactual labs](https://www.github.com/artefactual-labs), smaller projects and experiments
+- [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects
```
Comment (eviau-artefactual) [inline profile/README.md:16]:
```suggestion
- [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects
- [Artefactual-SDPS](https://github.com/artefactual-sdps) for projects related to Enduro
```

**Comment by jhsimpson** [comment]
looks good to me 👍

**Review by fiver-watson — APPROVED** [review]
:+1:
